packages/webdriverio/src/commands/browser/waitUntil.ts

```diff
@@ -16,7 +16,7 @@ import type { WaitUntilOptions } from '../../types.js'
  * @return {Boolean} true if condition is fulfilled
  * @uses utility/pause
  * @example https://github.com/webdriverio/example-recipes/blob/0bfb2b8d212b627a2659b10f4449184b657e1d59/waitUntil/index.html#L3-L8
- * @example https://github.com/webdriverio/example-recipes/blob/0bfb2b8d212b627a2659b10f4449184b657e1d59/waitUntil/waitUntilExample.js#L6-L14
+ * @example https://github.com/webdriverio/example-recipes/blob/main/waitUntil/waitUntilExample.js#L16-L24
```

Comment:
Let's not use `main` as branch name and rather pick a sha of your commit so we make sure the example never changes even though the example code does